crates/web5/src/dids/did.rs

```diff
@@ -115,6 +111,8 @@ mod tests {
     use super::*;
 
     mod new {
+        use std::sync::LazyLock;
+
```

Comment:
Tests don't affect MSRV

Reply:
oh interesting, this is still used? was it in preview for 1.70.0 or something and so it's okay inside of tests?

Reply:
we commented at the same time

so but even though our rustup is set to 1.70.0 in our Justfile, this still works?

Reply:
Good catch. Updated Justfile back to 1.80.0 and we'll just have the MSRV check run separately. Makes for an easier dev environment for the places where we don't need to worry about MSRV

Reply:
I kind of feel like we should just full commit to whatever our MSRV is, even in our local dev env, such that we don't develop only to find out that we have an incompatible feature until the CI pipeline is ran. Or would the `rust-version` property in the `Cargo.toml` already enforce that? Like if I have 1.80.0 in my local env, then I set the `rust-version` to 1.70.0 in my `Cargo.toml` and then I use a `LazyLock` in non-test code, and I run `cargo build` will it fail?

Reply:
> Like if I have 1.80.0 in my local env, then I set the rust-version to 1.70.0 in my Cargo.toml and then I use a LazyLock in non-test code, and I run cargo build will it fail?

It will not. It's only after the MSRV check that it would get flagged.

> we should just full commit to whatever our MSRV is

Word, let me try it

Reply:
We install packages during CI that require above 1.70.0. If we want to run everything on 1.70.0, not just the web5 crate where it is useful to lower MSRV, then we'll need to refactor those aspects of the CI and remove LazyLock from our tests. @KendallWeihe do you still want to use 1.70.0 for the entire codebase?

Reply:
I think my proposal is we go with 1.74.0 for right now https://github.com/TBD54566975/web5-rs/pull/303

And then let's open a ticket on the backlog to see if we can push it further lower at a later date